cloud/pkg/edgecontroller/controller/downstream.go

```diff
@@ -120,7 +121,7 @@ func (dc *DownstreamController) syncConfigMap(stop chan struct{}) {
 				default:
 					// unsupported operation, no need to send to any node
 					klog.Warningf("config map event type: %s unsupported", e.Type)
-					break
+					break nodeLoop
```

Comment:
great , but  this will affect the readability of the code. how about move `switch` out of `for loop`?

such as
```
		select {
		case e := <-dc.configmapManager.Events():
			configMap, ok := e.Object.(*v1.ConfigMap)
			if !ok {
				klog.Warningf("object type: %T unsupported", configMap)
				continue
			}

			operation := ""
			switch e.Type {
			case watch.Added:
				operation = model.InsertOperation
			case watch.Modified:
				operation = model.UpdateOperation
			case watch.Deleted:
				dc.lc.DeleteConfigMap(configMap.Namespace, configMap.Name)
				operation = model.DeleteOperation
			default:
				// unsupported operation, no need to send to any node
				klog.Warningf("config map event type: %s unsupported", e.Type)
				continue // continue to next select
			}

			nodes := dc.lc.ConfigMapNodes(configMap.Namespace, configMap.Name)

			klog.Infof("there are %d nodes need to sync config map, operation: %s", len(nodes), e.Type)
			for _, n := range nodes {
				...
				resource, err := messagelayer.BuildResource(n, configMap.Namespace, model.ResourceTypeConfigmap, configMap.Name)
				...
				msg.BuildRouter(constants.EdgeControllerModuleName, constants.GroupResource, resource, operation)
				...
				err = dc.messageLayer.Send(*msg)
				...
			}


```